frontend/eda/access/roles/EdaRoles.tsx

```diff
@@ -86,17 +60,21 @@ export function EdaRolesTable() {
     return filters;
```

Comment:
We may need to update the toolbar filters. Currently it brings up a list of all the roles as a dropdown.
`frontend/hub/access/roles/hooks/useRoleFilters.tsx` can be used as a reference here to provide a filter on Role name and editability.